src/db/common/SerializableVec.h

```diff
@@ -0,0 +1,22 @@
+#include <mw/serialization/Serializer.h>
+
+class SerializableVec : public Traits::ISerializable
+{
+public:
+    SerializableVec(std::vector<uint8_t>&& bytes) : m_bytes(std::move(bytes)) {}
+
+    std::vector<uint8_t> Get() const { return m_bytes; }
```

Comment:
nit: Return a `const std::vector<uint8_t>&` so it doesn't have to keep copying the vector